test_scripts/API/VehicleData/Refactor_Fuel_Information/common.lua

```diff
@@ -0,0 +1,159 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
I don't think that this vehicle data item should have its own separate common file and it should instead use commonVehicleData.lua